packages/headless/src/features/result-preview/result-preview-slice.ts

```diff
@@ -5,9 +5,16 @@ import {getResultPreviewInitialState} from './result-preview-state';
 export const resultPreviewReducer = createReducer(
   getResultPreviewInitialState(),
   (builder) => {
-    builder.addCase(
-      fetchResultContent.fulfilled,
-      (_, action) => action.payload
-    );
+    builder
+      .addCase(fetchResultContent.pending, (state) => {
```

Comment:
does calling `fetchResultContent` a second time reject the previous call if it's still pending?

Reply:
From what I am seeing in the chrome network tab, both requests are being resolved in order.